core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -47,39 +47,73 @@
     <background-banner></background-banner>
     <div ng-if="collection" class="oppia-collection-player-tiles-container">
       <h2 ng-if="!collectionPlaythrough.hasFinishedCollection()" class="oppia-page-heading">
-        <span ng-if="!collectionPlaythrough.hasStartedCollection()">Begin <[collection.getTitle()]>:</span>
+        <span ng-if="!collectionPlaythrough.hasStartedCollection()" style="font-family: Capriola, Roboto, Arial, sans-serif;">Begin <[collection.getTitle()]>:</span>
         <span ng-if="collectionPlaythrough.hasStartedCollection()">Continue <[collection.getTitle()]>:</span>
       </h2>
       <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-heading">
         <span>You have finished the collection! Feel free to replay any explorations below.</span>
       </h2>
 
       {% if is_logged_in %}
-        <div ng-if="collectionPlaythrough.getNextRecommendedCollectionNodeCount() > 0" class="oppia-collection-player-tile-section">
-          <div class="oppia-collection-player-small-text">
-            Next exploration<span ng-if="collectionPlaythrough.getNextRecommendedCollectionNodeCount() > 1">s</span> to play
-          </div>
-          <collection-node-list collection-id="collectionId" collection-nodes="getNextRecommendedCollectionNodes()"></collection-node-list>
-        </div>
+      <div ng-if="collection" style="height: 100%; margin-left: -205px; margin-right: auto; max-width: 675px; margin-top: 20px;
+      padding-bottom: 25px; padding-left: 20px; position: relative; background: #fff; border-radius: 2px;
+      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.14), 0 2px 4px rgba(0, 0, 0, 0.23); padding-top: 20px; text-align: left; float: left;">
+
+        <table class="oppia-dashboard-table">
+          <tr>
+            <th>
+              <p>Exploration</p>
+            </th>
+            <th>
+              <p>Objective</p>
+            </th>
+          </tr>
+          <tr ng-repeat="collection in collection.getCollectionNodes()"
+          class="exploration-list-item">
+          <td>
+            <a ng-href="/explore/<[collection._explorationId]>"
+              ng-mouseover="updateExplorationPreview(collection._explorationId)"
```

Comment:
Our policy for html is to align attrs on subsequent lines with the attr on the first line (look at other files to see how this is done). E.g.:

```
            <a ng-href="/explore/<[collection._explorationId]>"
               ng-mouseover="updateExplorationPreview(collection._explorationId)"
               ...
```

Note how the ng's line up. Please fix here and elsewhere.